custom/project_scrum_agile_extended/README.rst

```diff
@@ -0,0 +1,15 @@
+============================
+Project Scrum Agile Extended
+============================
+
+
+This module allows maintaining project Scrum Management Features specific for Onesetin's need.
```

Comment:
It was previously for onesetin, but features are depended with scrum extended, so instead of renaming module, we will update the description in readme.rst file